manticore/utils/helpers.py

```diff
@@ -34,6 +34,40 @@ def istainted(arg, taint=None):
     return False
 
 
+def get_taints(arg, taint=None):
+    '''
+    Helper to list an object taints.
+    :param arg: a value or Expression
+    :param taint: a regular expression matching a taint value (eg. 'IMPORTANT.*'). If None this functions check for any taint value.
+    '''
+
+    if not issymbolic(arg):
+        raise StopIteration
+    for arg_taint in arg.taint:
+        if taint is not None:
+            m = re.match(taint, arg_taint, re.DOTALL | re.IGNORECASE)
+            if m:
+                yield arg_taint
+        else:
+            yield arg_taint
+    raise StopIteration
+
+
+def taint_with(arg, taint, value_bits=256, index_bits=256):
+    '''
+    Helper to taint a value, Fixme this should not taint in place.
+    :param arg: a value or Expression
+    :param taint: a regular expression matching a taint value (eg. 'IMPORTANT.*'). If None this functions check for any taint value.
+    '''
+    if not issymbolic(arg):
```

Comment:
This check looks to be redundant; can just be the check on line below.

Reply:
arg is changed if in or long.

Reply:
`numbers.Integral` is the way to go imho, but we also have `isint()` in the py3 branch which we can port